interface/wx/translation.h

```diff
@@ -581,10 +581,10 @@ class wxMsgCatalog
 
     This function is thread-safe.
 
-    @note This function is not suitable for literal strings using wxT() macro
+    @note This function is not suitable for literal strings
           since this macro is not recognised by @c xgettext, and so such
           strings are not extracted to the message catalog. Instead, use the
-          _() and wxPLURAL() macro for all literal strings.
+          _() and wxPLURAL() macros for all literal strings.
 
     @see wxGetTranslation(const wxString&, const wxString&, unsigned, const wxString&)
```

Comment:
BTW, this is not linked in [the generated docs](https://docs.wxwidgets.org/latest/group__group__funcmacro__string.html#ga49dcdc17d82a627b7a50d401f1a37d69) and I wonder if this could be fixed by using a correct signature (adding the context argument):
```suggestion
    @see wxGetTranslation(const wxString&, const wxString&, unsigned, const wxString&, const wxString &)
```
[The other override description](https://docs.wxwidgets.org/latest/group__group__funcmacro__string.html#ga3fca2c2319d2481568aef51678e8021e) suffers from the same problem, see the first line.

Reply:
Thanks, PB. I saw a few more missing links, so I fixed those too.